### DISCUSSION TITLE
Deep-Link Support

### DISCUSSION THREAD

**Discussion by yunusemrecetin** [nylo-core/nylo#49] [General]
Currently, Framework does not support the deep link. Did you have any plans for implementing it?

**Comment by agordn52** [comment]
Hi @yunusemrecetin, do you mean 'Universal Links'? 
If not, can you give an example, thanks

**Reply by HynRidge** [reply]
Dear @agordn52 , I also have the same concern. Deeplink is to allow user access our app from another app. For example, in instagram, there's an whatsapp link. When user click on this whatsapp link, it will open up whatsapp and open the chat screen of that user. From my understanding, Universal Link is for iOS, while deeplink is for AndroidOS. I'm just wondering if you're `NyRouter` 
module is supporting this capabilities as well. Thanks

**Reply by agordn52** [reply]
Yes, Deep linking is supported. Here's a clip of it working in Nylo.

https://github.com/nylo-core/nylo/assets/17294994/cf806c6e-c084-4e09-aa1f-9bc5997e5ced


You can follow this guide https://docs.flutter.dev/cookbook/navigation/set-up-universal-links#2-adjust-ios-build-settings here to get started, skip section 1. Thanks 👍
